recipe/build-sysroot.sh

```diff
@@ -33,4 +33,8 @@ ln -s $PWD/usr/lib64 $PWD/usr/lib
 cp "${SRC_DIR}"/binary-freebl/lib64/libfreebl3.so ${PWD}/lib64/.
 ln -s ${PWD}/lib64/libfreebl3.so ${PWD}/usr/lib64/libfreebl3.so
 
+sed -i.bak 's,AS_NEEDED,AS_NEEDED ( /lib64/librt.so.1 ) AS_NEEDED,g' usr/lib64/libc.so
+cat usr/lib64/libc.so
```

Comment:
We may want to remove this before merging.

Reply:
Appears to be resolved in commit ( https://github.com/conda-forge/linux-sysroot-feedstock/pull/50/commits/437bb5bdb8fe93c3794f36a29b9871ad96079394 ) below